.github/workflows/github_pages.yml

```diff
@@ -32,7 +32,7 @@ jobs:
 
       - name: Build docs
         run: |
-          pip install gitpython packaging toml Sphinx sphinx-rtd-theme==1.0.0
+          pip install gitpython packaging toml Sphinx sphinx-rtd-theme==1.0.0 datumaro
```

Comment:
This fix will use the released version. Probably, it is not valid in this situation. Consider using `pip install path/to/the/repo` (which should be the current directory on the CI).

Reply:
Looks like need modules from `requirements.txt`, not the `datumaro` module itself